src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,7 @@
 package core.basesyntax;
 
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public String getRandomColor(int random) {
+        return Colors.values()[random].toString();
```

Comment:
Now your CollorSupplier.getRandomBall returns not random color. It returns a color by index that it received in `random` variable.
Create a `Random` class instance for CollorSupplier and make it return a color with a random index.

Reply:
Also please, use `Color.name()` instead of `Color.toString()` or return the enum instance directly.